impl/src/main/java/com/okta/sdk/impl/resource/DefaultGroupRuleBuilder.java

```diff
@@ -22,13 +22,11 @@
 
 import java.util.ArrayList;
 import java.util.List;
-import java.util.Objects;
 
 public class DefaultGroupRuleBuilder implements GroupRuleBuilder {
 
     private String name;
     private String type;
-    private Boolean allGroupsValid;
```

Comment:
Sorry if I already asked this, but was this field removed?

Reply:
yes.